Makefile

```diff
@@ -27,25 +28,31 @@ install: clean
 	$(R) CMD INSTALL .
 
 uninstall:
-	$(R) --slave -e "remove.packages('instrumentr')"
+	$(R) -s -e "remove.packages('instrumentr')"
```

Comment:
slave is deprecated https://github.com/wch/r-source/commit/f1ff49e74593341c74c20de9517f31a22c8bcb04